tailwind.config.js

```diff
@@ -18,6 +18,10 @@ module.exports = {
         secondary: '#F4DCF4',
         'app-background': '#fff',
         orange: '#ff5a42',
+        listCategory: '#f5f5dc',
```

Comment:
call this workitem instead of listCategory. Similarly the next line

Reply:
changed It.